src/i18n/locales/zh-CN

```diff
@@ -0,0 +1,375 @@
+    },
+    "required_files": "必要文件",
+    "required_files_count": "{count} / {total} 文件",
+    "settings": "配置",
+    "sponsor": "Sponsor",
+    "supports": "Supports",
+    "supports_items": {
+      "cartridges": "卡带",
+      "dock": "底座", //do you mean the 99.99 USD Analogue Dock? If it is, the translation is correct
+      "dock_dac": "DAC解码器", // are u refering this item? https://www.analogue.co/dac
+      "save_states_and_sleep": "即时存档/睡眠"
+    },
+    "url": "网页地址",
+    "version": "版本"
+  },
+  "core_info_required_files": {
+    "close": "关闭",
+    "download_all": "下载全部",
+    "no_link_tip": "请进入配置页面查询必要文件的更多选项",
+    "skip_alternates_label": "跳过次要文件",
+    "title": "必要文件",
+    "file_status": {
+      "wrong": "可更新",
+      "downloadable": "可下载",
+      "not_in_archive": "未入库",
+      "ok": "已下载",
+      "unknown": "未知",
+      "at_root": "已存在于根目录"
+    }
+  },
+  "cores": {
+    "controls": {
+      "category": "类型",
+      "refresh": "刷新",
+      "search": "搜索",
+      "updatable": "可更新"
+    },
+    "install_tip": "您也可以通过拖拽ZIP压缩包到窗口内的方式安装核心或其他内容",
+    "installed": "已安装({count})",
+    "not_installed": "可安装({count})"
+  },
+  "error": {
+    "report": "您可以通过GitHub报告错误信息（含错误信息和错误发生的描述性文字）",
+    "retry": "重试",
+    "title": "错误"
+  },
+  "fetch": {
+    "controls": {
+      "add_fetch_item": "Add new fetch",
+      "refresh": "刷新"
+    },
+    "copying": "正在复制文件...",
+    "fetch_button": "Fetch",
+    "file_count": "{count} / {total, plural, one {# 文件} other {# 文件}}",
+    "loading": "载入中……",
+    "new": {
+      "add_extension": "添加",
+      "add_fetch": "Add fetch",
+      "archive_name": "Archive name: <i>https://archive.org/details/[this bit here]</i>",
+      "archive_tip": "This won't work for achives where the files are zipped, and you should set which extensions you want to avoid downloading the metadata files.",
+      "close": "关闭",
+      "destination": "目标目录",
+      "extensions": "限制文件扩展名(类型):",
+      "origin": "源目录",
+      "title": "Add new Fetch"
+    },
+    "remove_button": "移除",
+    "remove_confirm": "This'll remove the fetch configuration, but not the files. Are you sure?",
+    "types": {
+      "archive_org": "在线文件", //refering to online(local&internet) source, correct?
+      "filesystem": "本地文件"
+    }
+  },
+  "firmware": {
+    "current": "现有固件版本: v{version}",
+    "downloading_firmware": "下载固件中...",
+    "load_firmware": "上传至SD卡",
+    "loaded_firmware": "固件 {version} 已经上传至SD卡",
+    "loaded_firmware_info": "插入SD卡并启动Pocket完成新固件安装",
+    "no_firmware_loaded": "未发现SD卡上有待安装的固件",
+    "options": {
+      "latest": "v{version} (latest)",
+      "older": "v{version} ({date, date, short})",
+      "previous_versions": "上一版本"
+    },
+    "release_notes": {
+      "from": "来自 ",
+      "title": "v{version} 更新记录:"
+    },
+    "remove_button": "移除"
+  },
+  "games": {
+    "controls": {
+      "category": "类型",
+      "clean_files": "清理文件",
+      "instance_json": "Instance JSON", //don't know what does it mean...
```

Comment:
So some cores like PC Engine can build what are referred to as "Instance JSON" files where there's a .json file created which refers to other files (.bin & .cue in the case of PC Engine) and is what the user selects from the Pocket menu.

The Analogue Docs go into it a little here: https://www.analogue.co/developer/docs/core-definition-files/instance-json

Reply:
I tried to understand... Is it more like an instance config JSON file? Like a database instance?

Reply:
Sort of, yeah. It represents an "instance" of a game or whatever including the names & locations of all the files required to run that game.

Arcade games use it to define the locations of the .rom files & any other setup that each game requires, the PC Engine (CD) core uses it to represent all the data that's on the CD (the .bin & .cue files) - the NeoGeo core uses it to represent the 4 or 5 different files that need to be loaded to play a NeoGeo game.

And this option is used when the core defines methods for updaters to generate the Instance JSON files based on what existing files you have, currently it's just used by the PC Engine CD core, but the plan is for an upcoming version of the NeoGeo core to switch to it (rather than requiring people load a certain ROM pack & showing all the JSON files for games they might not have)